### DISCUSSION TITLE
TLS access on bitcoinRPC

### DISCUSSION THREAD

**Discussion by jhub-beep** [raspiblitz/raspiblitz#4380] [Q&A]
Hi there, I am searching for a way to access my node via bitcoinRPC with the use of TLS (1.3) on my LAN. Is that possible? I only got it without TLS working.

**Comment by rootzoll** [comment]
AFAIK Bitcoin Core does not natively support TLS for RPC connections. To enable TLS, you need to set up a proxy that listens to RPC calls and forwards them to Bitcoin Core over TLS. You might be able to use nginx for this, which we have already running as part of our RaspiBlitz WebUI and that is working as a HTTPS proxy for other additional apps.

Please share any progress you make on this if you try - could make sense to add this as a standard feature in the future.

**Comment by openoms** [comment]
Curious, what is the use case of this?
